### ISSUE TITLE
Deletion operations should ask for confirmation

### ISSUE THREAD

**Issue by martialblog** [Icinga/icinga-notifications-web#249]
Hi,

I think deleting Contacts, Event Rules, etc. should ask for a confirmation. 
Complex Event Rules are gone in an instant if not careful.

Cheers,
Markus

**Comment by nilmerg** [comment]
Since we don't really delete anything, I'd rather like to propose to introduce a restoration feature. 🙏

**Comment by nilmerg** [comment]
Nope. A confirmation dialog feature will be added to Icinga Web to facilitate here.
